anta/tests/bfd.py

```diff
@@ -22,12 +21,24 @@
 
 
 class VerifyBFDSpecificPeers(AntaTest):
-    """Verifies if the IPv4 BFD peer's sessions are UP and remote disc is non-zero in the specified VRF.
+    """Verifies if the IPv4 BFD peer sessions are in the UP state and the remote discriminator (disc) is non-zero within the specified VRF.
```

Comment:
Since we are now describing what the test checks below, we can simplify this.
```suggestion
    """Verifies the state of IPv4 BFD peer sessions.
```

Reply:
Updated.